browser/branding.h

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: 2023 Robin Lindén <dev@robinlinden.eu>
+//
+// SPDX-License-Identifier: BSD-2-Clause
+
+#ifndef BROWSER_BRANDING_H
```

Comment:
Nit-pick, but I would have called the module "version" or "constants" or something like that. Ultimately doesn't really matter.